src/explorer/SiteTreeItem.ts

```diff
@@ -182,4 +195,15 @@ export abstract class SiteTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
             telemetryProperties.promptToSaveDeployConfigs = 'Skip for now';
         }
     }
+
+    private getIgnoredFoldersForDeployment(runtime: string): string[] {
+        switch (runtime) {
+            case runtimes.node:
+                return ['node_modules{,/**}'];
+            case runtimes.python:
+                return ['env/{,/**}', 'venv{,/**}', 'lib{,/**}', 'lib64{,/**}'];
```

Comment:
You keep changing this list - how are you picking these??

Reply:
Just looking at a .gitignore file on GitHub and picking ones that seem relevant (like pip installs into a Lib folder, as far as I can tell) https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
I would prefer that all envs listed in the gitignore were included here before checking in (on the off chance that we don't implement the "auto-detect" before releasing)
aka all of these:
```
# Environments
.env
.venv
env/
venv/
ENV/
env.bak/
venv.bak/
```